pkg/client/rest_client.go

```diff
@@ -0,0 +1,29 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	authv1 "k8s.io/api/authentication/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/rest"
+	"k8s.io/utils/pointer"
+)
+
+// CreateTokenRequest creates a TokenRequest for a service account using given expiration in seconds.
```

Comment:
Please also indicate that the function returns the generated token.

Reply:
fixed the comment in https://github.com/codeready-toolchain/toolchain-common/pull/310/commits/140156e24887adc4521460abff285b48166fd1da